{{cookiecutter.project_name}}/{{cookiecutter.app_name}}/api/resources/user.py

```diff
@@ -1,22 +1,12 @@
 from flask import request
 from flask_restful import Resource
 from flask_jwt_extended import jwt_required
-
+from {{cookiecutter.app_name}}.api.schemas.user import UserSchema
```

Comment:
Since you're re-exporting `UserSchema` in `api/schemas/__init__.py` you should be able to reduce this import to `from {{cookiecutter.app_name}}.api.schemas import UserSchema`

Reply:
Yeah, you are right on this....
I will check it out

Reply:
@karec , I am done with refractoring , let me know if there is something  else I can improve.